plugins/modules/iosxr_command.py

```diff
@@ -34,6 +33,8 @@
       The resulting output from the command is returned. If the I(wait_for) argument
```

Comment:
Adding a quick note about the variation of elements in this list may be worth it. For e.g., https://github.com/ansible-collections/cisco.nxos/pull/126/files#diff-8edd2850a1c49131fc5d71280acd74bbR32 (copied from ios_config)

Reply:
yes, that would be good, I'll update this. thanks